docs/pyproject.toml

```diff
@@ -0,0 +1,16 @@
+[project]
+name = "niri"
+version = "0.1.0"
+description = "Add your description here"
```

Comment:
Is this field required?

Reply:
Unsure, probably not. I'll look at it soontm